components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -680,7 +680,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             return;
         }
         let now = Instant::now();
-        for i in from + 1..to {
+        for i in from + 1..=to {
```

Comment:
I think it belongs the type/bug, you should add some comment on the pr.

Reply:
Done